brainscore/metrics/transformations.py

```diff
@@ -311,6 +317,13 @@ def pipe(self, source_assembly, target_assembly):
         split_scores = []
         for split_iterator, (train_indices, test_indices), done \
                 in tqdm(enumerate_done(splits), total=len(splits), desc='cross-validation'):
+            
+            if hasattr(self, '_preprocess_indices'):
+                if self._preprocess_indices is not None:
+                    print('train_indices before: ', len(train_indices))
+                    train_indices, test_indices = self._preprocess_indices(train_indices, test_indices, source_assembly)
+                    print('train_indices after: ', len(train_indices))
```

Comment:
```suggestion
            if self._preprocess_indices is not None:
                train_indices, test_indices = self._preprocess_indices(train_indices, test_indices, source_assembly)
```
no prints. If these are important to log, use a `logger`.